src/docs/self-hosted/index.mdx

```diff
@@ -8,6 +8,42 @@ In addition to making its source code available publicly, Sentry offers and main
 
 Our recommendation is to download the [latest release of the self-hosted repository](https://github.com/getsentry/self-hosted/releases/latest), and then run `./install.sh` inside this directory. This script will take care of all the things you need to get started, including a base-line configuration, and then will tell you to run `docker-compose up -d` to start Sentry. Sentry binds to port `9000` by default. You should be able to reach the login page at [http://127.0.0.1:9000](http://127.0.0.1:9000/).
 
+### Installing Behind a Proxy
+
+In some enterprise setups there is no direct Internet connection, you must use an HTTP proxy server. How do you install Sentry in this environment? Let us assume: 
+
+1. Your Sentry installation is running on Linux.
+1. `http://proxy:3128` is your proxy address.
+1. `127.0.0.0/8` is the only network that should be accessed without a proxy.
+
+Here are the steps to follow:
+
+1. Set `http_proxy`, `https_proxy` and `no_proxy` variables in the `/etc/environment` file.
+2. To make the `docker pull` command respect your proxy settings, create a `/etc/systemd/system/docker.service.d/http-proxy.conf` file with these contents:
+```
+[Service]
+Environment="HTTP_PROXY=http://proxy:3128"
+Environment="HTTPS_PROXY=http://proxy:3128"
```

Comment:
```suggestion
Environment="HTTPS_PROXY=https://proxy:3128"
```

I definitely could be mistaken, but I believe `HTTPS_PROXY` *must* be set to an `https` URL.

Reply:
If I'm not mistaken `HTTP_PROXY` variable is used for HTTP requests and `HTTPS_PROXY` - for HTTPS requests (found in [curl man page](https://curl.se/docs/manpage.html)). So it's possible to use http/https proto for HTTP_PROXY and for HTTPS_PROXY.

Reply:
Yup, that definitely seems to be correct, I will resolve these reviews then.

Reply:
Hmmm ... reading the curl docs I see "http_proxy is an exception as it is only available in lower case." Do we need to change that here? Maybe suggest setting both?

Reply:
It seems according to the [docker docs](https://docs.docker.com/network/proxy/#set-the-environment-variables-manually) that the upper case usage is correct.